conjure-undertow-lib/src/main/java/com/palantir/conjure/java/undertow/lib/Endpoint.java

```diff
@@ -111,6 +120,12 @@ public Builder deprecated(Optional<String> value) {
             return this;
         }
 
+        @CanIgnoreReturnValue
+        public Builder deprecated(List<String> value) {
```

Comment:
s/deprecated/tags/

Reply:
Good call. Should take an `Iterable` as well!